### ISSUE TITLE
dependencies are out of date

### ISSUE THREAD

**Issue by yegor256** [yegor256/rultor#946]
Too many deps are out of date, let's fix that, see https://www.versioneye.com/user/projects/561a9d87a193340f28000fd3

**Comment by alex-palevsky** [comment]
@yegor256 added "bug" tag to this issue

**Comment by alex-palevsky** [comment]
@yegor256 milestone set to `2.0` (correct me if I am wrong)

**Comment by alex-palevsky** [comment]
@yegor256 thank you for reporting this, I added 15 mins to your acc, transaction `000-edc0637a`

**Comment by alex-palevsky** [comment]
@jimdeanspivey please go ahead, it's your task now, keep [this](http://www.xdsd.org/2014/04/17/how-xdsd-is-different.html) in mind, and don't hesitate to ask any technical questions you may have. Task's budget is **30 mins** (see [this](http://www.xdsd.org/2014/04/17/how-xdsd-is-different.html) for explanation)

**Comment by JimDeanSpivey** [comment]
@yegor256 

1. xembly has changed from `0.21.4` to `0.22`. This dependency is needed in jcabi-github so it has to be changed in both projects at the same time in order for rultor to compile. **Should I leave it in the older version or update both/all projects that use xembly?** For reference, the API docs are here, [0.21.4](http://www.xembly.org/apidocs-0.21.4/org/xembly/class-use/Directives.html) and [0.22](http://www.xembly.org/apidocs-0.22/org/xembly/class-use/Directives.html).  You can see a change in how .22 accept `Object` now instead of `String` and this will cause a NoSuchMethodException if dependencies try to use different jars than what other compiled dependencies are trying to use (eg: jacbi-github).
2. Similar to number 1, takes `.29` uses the older `0.21.4` xembly but takes `.31` uses the nwer `0.22` version of xembly. Therefore, upgrading to takes *and not* upgrading to xembly `0.22` will also fail the build. So takes was not upgraded in the incoming pull request.

Basically, should I get pull request rights and upgrade xembly in jcabi-github project? Or should a pull request with no upgrades to takes and xembly be accepted in this issue? All the other upgrades run fine.

**Comment by yegor256** [comment]
@JimDeanSpivey you should submit tickets to other projects and wait until they are resolved

**Comment by yegor256** [comment]
@JimDeanSpivey this should help: http://www.yegor256.com/2015/01/15/how-to-cut-corners.html

**Comment by JimDeanSpivey** [comment]
@yegor256 There is already a ticket in jacbi-github, I am asking to work on it here: https://github.com/jcabi/jcabi-github/issues/1188

**Comment by alex-palevsky** [comment]
@jimdeanspivey the ticket is with you for 15 days already. If it is not finished in the next 48 hours, it will be re-assigned to someone else, see [No Obligations](http://www.xdsd.org/2014/04/13/no-obligations-principle.html) principle, [this article](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html) should help if you're stuck... -30 added to your rating, at the moment it is: [+38](http://www.netbout.com/b/36321?open=rating)

**Comment by alex-palevsky** [comment]
@jimdeanspivey the task is overdue, and I have to re-assign it to someone else. Please, stop working with it immediately. In general, we're against overdue tasks, check [this page](http://www.xdsd.org/2014/04/13/no-obligations-principle.html). -60 to your rating, your total score is [-52](http://www.netbout.com/b/36321?open=rating)

**Comment by original-brownbear** [comment]
#1005 should do it here.
